testsuite/gnat2goto/tests/arrays_attributes/arrays_attributes.adb

```diff
@@ -0,0 +1,17 @@
+procedure Arrays_Attributes is
+   subtype Range1 is Integer range 10 .. 20;
+   type Arr3 is array (Range1) of Integer;
+   Actual3 : Arr3 := (7, 8, 9, others => 10);
+   subtype My_Index is Integer range Actual3'Range;
+   Array_Attribs : Integer;
+   Val : Integer := 10;
+   An_Index : My_Index := 12;
+begin
+   Actual3 (10 .. 11) := Actual3 (17 .. 18);
+   Actual3 (11) := 100;
+   Actual3 (13 .. 16) := Actual3 (11 .. 12) & Actual3 (15 .. 16);
+   Array_Attribs := Actual3'First + Actual3'Last + Actual3'Length;
+
+   pragma Assert (Array_Attribs = 41);
+   pragma Assert (An_Index > 7);
```

Comment:
An_index is not reassigned to so seeing this check not as =12 made me check twice if it was assigned to somewhere I'd missed.

Reply:
Done.